azurerm/data_source_express_route_circuit_test.go

```diff
@@ -0,0 +1,83 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+)
+
+func TestAccDataSourceAzureRMExpressRoute_basic(t *testing.T) {
```

Comment:
since there's a limit on the number of ExpressRoute's which can be provisioned at one time (and we run the tests in parallel) - we'll need to make this an internal method (by making it lower-case) and then ensure it's called from [within this test function](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_express_route_circuit_test.go#L15-L41) in the Express Route resource

(but to make this simpler to test, we probably want to do this when all the other changes are done 😄)

Reply:
Since it only has the 1 test in there, do we need the function? Also, should add this function to the [test function](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_express_route_circuit_test.go#L15-L41) (in the resource file) or should we create a similar function in this test file?